### PR TITLE
sdk/node: tick version to 1.1.0

### PR DESCRIPTION
**Pull request by jeffomatic**

### CODE REVIEW

**Comment by jeffomatic** [comment]
pending additional source docs

File: sdk/node/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,150 @@
...
+  ...
+})
+```
+
+##### New (1.1.x)
+
+```
+templatePromise = client.transactions.build(builder => {
+  builder.controlWithReceiver({
+    reciever: receiver,
```
Comment (chrisgarvin) [inline sdk/node/CHANGELOG.md]: `receiver: receiver`

File: sdk/node/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,150 @@
...
+Transaction output objects and unspent outputs now have an `id` property, which is unique for that output across the history of the blockchain.
+
+```
+console.log(tx.outputs[0].id)
+console.log(utxo.id)
+```
+
+##### Transaction inputs
+
+The `spentOutput` property on transaction intputs is **deprecated**. Use `spentOutputId` instead.
```
Comment (chrisgarvin) [inline sdk/node/CHANGELOG.md]: `inputs`

File: sdk/node/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,150 @@
...
+
+```
+console.log(tx.inputs[0].spentOutputId)
+```
+
+#### Spending unspent outputs in transactions
+
+The `spendUnspentOutput` method now takes an `outputId` parameter. The `transactionId` and `position` parmeters are **deprecated**.
```
Comment (chrisgarvin) [inline sdk/node/CHANGELOG.md]: `parameters`

File: sdk/node/README.md [hunk 4]
```diff
@@ -51,8 +47,7 @@ client.transactions.query({}, callback)
 
 ## Using external signers
 
-To connect to an HSM other than the built-in Mock HSM, you must create a new
-`Connection` object:
+To connect to an HSM other than the built-in Mock HSM, you must create a new `Connection` object:
```
Comment (chrisgarvin) [inline sdk/node/README.md]: Should the naming here be `MockHSM` as we have it named elsewhere?

Comment (jeffomatic) [inline sdk/node/README.md]: Yep

**Pull request merged**